modules/ROOT/pages/mongodb/mongodb-connector-6-0-studio.adoc

```diff
@@ -0,0 +1,100 @@
+= MongoDB Studio Configuration - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+MongoDB Connector Version 6.0
+
+To configure a connector in Anypoint Studio follow this process:
+
+. Install the connector.
```

Comment:
Should we add "...connector using Exchange or Studio"?

Reply:
Leaving as is for now. In our revised template (in the works), we will show just one way to install, so this will become moot.